asb-ballerina/messageReceiver.bal

```diff
@@ -31,14 +31,11 @@ public isolated client class MessageReceiver {
     # obtain tokens following [this guide](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string). 
     # Configure the connection string to have the [required permission](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas).
     # 
-    # + connectionString - Connection String of Azure service bus
-    # + entityPath - Name or path of the entity (e.g : Queue name, Subscription path)
-    # + receiveMode - Receive mode as PEEKLOCK or RECEIVEANDDELETE (default : PEEKLOCK)
-    public isolated function init(@display {label: "Connection String"} string connectionString, @display {label: "Entity Path"} string entityPath, 
-                                  @display {label: "Receive Mode"} string receiveMode = PEEKLOCK) returns error? {
-        self.connectionString = connectionString;
-        self.entityPath = entityPath;
-        self.receiveMode = receiveMode;
+    # + config - Connection config of ASB Receiver
+    public isolated function init(ConnectionConfig config) returns error? {
```

Comment:
@SkNuwanTissera Let's revert this change. But still we need to update Dependencies.toml and bump minor version